doc/nrf/ug_radio_coex.rst

```diff
@@ -150,20 +214,23 @@ This implementation uses the following pins to communicate with the PTA:
 * PRIORITY pin - It is the output controlled by the protocol driver.
   It contains information about the type of operation (either RX or TX) to perform.
   It can be configured by setting the ``pri-dir-gpios`` property of the ``nrf_radio_coex`` devicetree node.
-* GRANT pin - It is the input of the SoC controlled by the PTA.
+* GRANT pin - It is the input of the SoC controlled:kconfig:option:`CONFIG_MPSL_CX` by the PTA.
```

Comment:
This change seems to be unwanted (looks like an accidental insertion).

Reply:
fixed